jupyterlab_automl/src/components/list_resources_panel.tsx

```diff
@@ -417,32 +526,37 @@ export class ListResourcesPanel extends React.Component<Props, State> {
   }
 
   private async refresh() {
-    try {
-      this.setState({ isLoading: true });
-      await Promise.all([this.getDatasets(), this.getModels()]);
-      this.setState({ hasLoaded: true });
-    } catch (err) {
-      console.warn('Error retrieving datasets', err);
-    } finally {
-      this.setState({ isLoading: false });
-    }
+    this.setState({ isLoading: true });
+    await Promise.all([
```

Comment:
Since you swallow any errors in the get* methods, there's no great way for a user to tell if something didn't refresh properly. Not a big issue for an MVP, but you may want to add a TODO to give some consideration to what can be surfaced if data doesn't get retrieved successfully.